target/linux/ipq40xx/config-4.14

```diff
@@ -102,6 +102,7 @@ CONFIG_CPU_IDLE_MULTIPLE_DRIVERS=y
 CONFIG_CPU_PABRT_V7=y
 CONFIG_CPU_PM=y
 CONFIG_CPU_RMAP=y
+CONFIG_CPU_SPECTRE=y
```

Comment:
This raises a question though. The A7 is an Partial dual-issue in-order CPU it should be safe from Spectre at least according to:

https://developer.arm.com/support/arm-security-updates/speculative-processor-vulnerability

and https://github.com/speed47/spectre-meltdown-checker 

should we still set it or save some cycles?

Reply:
I can give it a test with that checker tomorrow.
But I think that it is in sync with the ARM publication, so if ARM states it's not vulnerable then we can unset the flag and save some resources.

This reminds me that 4.19 also has Spectre flag set per generic configuration.

Reply:
Ok, let's continue tomorrow.

Reply:
Awesome.
Here is what that spectre-meltdown checker reports:
https://gist.github.com/robimarko/bdb7a69704207a67aef77c651ec5f471

Basically everything is stating that it is not vulnerable and it looks like kernel is not using IBRS at all so we can leave the support compiled in and it wont make any difference.

Reply:
Yeah, it makes sense since its not used anyway.
Will do it soon

Reply:
@chunkeey It looks like CONFIG_CPU_SPECTRE cant be easily disabled.
As soon as CONFIG_CPU_V7 is selected it will autoselect CONFIG_CPU_SPECTRE and even if you disable it manually it will get set again.
I dont think that its worth having another patch, so we can just leave it as is since its not used anyway and has no performance effect.